VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-info-tooltip/participant-info-tooltip.component.html

```diff
@@ -14,7 +14,7 @@
     <p id="tooltip-joined-by-quick-link-{{ participant.id }}" class="govuk-!-margin-bottom-1">{{ joinByQuickLinkText }}</p>
   </div>
   <div class="govuk-grid-column-full">
-    <p *ngIf="participant.showCaseRole" id="tooltip-case-role-{{ participant.id }}" class="govuk-!-margin-bottom-1">
```

Comment:
What is the purpose of having data-case-role as an attribute? It appears to just be used in the test as a selector is that correct?